### PR TITLE
providers: added 2, removed 1, updated 1

### PR DESCRIPTION
**Pull request by jospoortvliet**
Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@MariusBluem @tflidd @splitt3r any chance one of you can have a quick look? Shouldn't be too scary...

File: assets/providers.json [hunk 1]
```diff
@@ -1140,5 +1127,34 @@
...
+         "freeplans":false,
+         "supports":"both",
+         "specializes":[
+            "We offer hosted, Vps, On Premise Nextcloud and Collabora solutions. We develop apps and core customization."
+         ],
+         "imagename":"semantica.png"
+      },
+    {
+         "title":"Infmaster",
+         "url":"https:\/\//nextcloud.infmaster.com.br/",
```
Comment (splitt3r) [inline assets/providers.json]: Should be `https:\/\/nextcloud.infmaster.com.br\/`?

File: assets/providers.json [hunk 2]
```diff
@@ -1140,5 +1127,34 @@
             "DediSERVE are the world largest OnApp provider, specializing in blazing fast All-SSD public and private cloud environments with 17 locations worldwide"
          ],
          "imagename":"dediserve.png"
+      },
+    {
+         "title":"Semantica",
+         "url":"https:\/\//s7a.it\/web\/products_cloudsystem",
```
Comment (splitt3r) [inline assets/providers.json]: Should be `https:\/\/s7a.it\/web\/products_cloudsystem`?

**Comment by jospoortvliet** [comment]
Good catches, fixed!

**Pull request merged**